relay_commander/version.py

```diff
@@ -6,7 +6,7 @@
 
 .. autodata:: VERSION
 
-.. versionadded:: 0.0.12
+.. versionadded:: 0.0.13
```

Comment:
This should refer to the version where this function was added for documentation purposes, there is no need to ever increment this.

Reply:
Should I be updating the rest of `versionchanged` and `versionadded` appropriately for documentation purposes?